scripts/update-remote-wn-client

```diff
@@ -155,6 +155,7 @@ def rsync_upload(local_dir, remote_user, remote_host, remote_dir, ssh_key=None):
     except OSError as e:
         raise Error(errstr + str(e))
     output, _ = proc.communicate()
+    output = output.decode()
```

Comment:
```suggestion
    output = output.decode("latin-1")
```
just to make sure we don't get a UnicodeDecodeError in case of invalid characters in the output.  Alternatively, `output.decode("utf-8", errors="<ANYTHING BUT STRICT>")`

Reply:
Yeah normally i'd either say something like "well why _shouldn't_ we allow non-ascii unicode?", or "why _shouldn't_ we fail miserably if there's something nasty in the output?" - but in this case we just care about comparing to the string `"missing"` so i guess no harm really done in making the glass slipper fit.